pom.xml

```diff
@@ -1825,6 +1825,16 @@
         <artifactId>mbknor-jackson-jsonschema_${scala.binary.version}</artifactId>
         <version>1.0.39</version>
       </dependency>
+      <dependency>
+        <groupId>io.netty</groupId>
+        <artifactId>netty-common</artifactId>
+        <version>4.1.96.Final</version>
```

Comment:
Define `<netty.verssion>` in `<properties>` instead of hardcoding the versions here.

Reply:
Done.